modules/configuration/src/main/java/org/apache/ignite/internal/configuration/DynamicConfiguration.java

```diff
@@ -100,8 +100,12 @@ protected DynamicConfiguration(
         return refreshValue();
     }
 
-    /** {@inheritDoc} */
-    @Override public Map<String, ConfigurationProperty<?, ?>> members() {
+    /**
+     * Returns all child nodes of current configuration tree node.
+     *
+     * @return Map from {@code String} to a corresponding {@link ConfigurationProperty}.
```

Comment:
what does this `String` represent in this context?

Reply:
sorry